README.rst

```diff
@@ -82,6 +95,32 @@ warnings=True|False
    :width: 40pt
 
 
+**Colorization:**
+
++------------------------------------------------+-------------------------------------------------------------------+
+| ``LayerViewDigraph(lv, ...)``                  |         ``render(...)``                                           |
++--------------+-------------+-------------------+-------------------------------------------------------------------+
+| ``compact=`` | ``colors=`` | ``color_mode=``   |      ``rankdir='BT'``                                             |
++==============+=============+===================+===================================================================+
+| *True*       | **4**       | *breadth*         | |BT_compact_with-terms_warnings-disabled_4-colors-by-breadth|     |
++--------------+-------------+-------------------+-------------------------------------------------------------------+
+| *True*       | **4**       | **depth**         | |BT_not-compact_with-terms_warnings-disabled_4-colors-by-depth|   |
```

Comment:
This one should be `|BT_compact_with-terms_warnings-disabled_4-colors-by-depth|`